infinitic-storage/src/main/kotlin/io/infinitic/storage/config/mysql/MySQLKeyValueStorage.kt

```diff
@@ -41,9 +41,12 @@ class MySQLKeyValueStorage(internal val pool: HikariDataSource) : KeyValueStorag
       connection
           .prepareStatement(
               "CREATE TABLE IF NOT EXISTS $MYSQL_TABLE (" +
-                  "`id` INT AUTO_INCREMENT PRIMARY KEY," +
+                  "`id` BIGINT(20) AUTO_INCREMENT PRIMARY KEY," +
                   "`key` VARCHAR(255) NOT NULL UNIQUE," +
-                  "`value` BLOB NOT NULL" +
+                  "`value` LONGBLOB NOT NULL" +
+                  "`last_update` timestamp NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP," +
+                  "`value_size_in_ko` BIGINT(20) GENERATED ALWAYS AS ((length(`value`) / 1024)) STORED," +
```

Comment:
Although octet is a valid word, bytes is way more commonly used especially in english. 
So I think we should use bytes, also if you're dividing by 1024, then it would be more accurate to name the unit `KiB` as you would store Kibibytes and not Kilobytes technically (1 KiB = 1024 bytes, 1 kB = 1000 bytes). 

Maybe dividing by 1000 and naming it kB could also be better as kB are more commonly used to talk about storage & disk space then KiB